gwcli/storage.py

```diff
@@ -407,13 +406,38 @@ def ui_command_resize(self, image_id, size):
         disk = self.disk_lookup[image_id]
         disk.resize(size)
 
+    def ui_command_snapshot(self, image_id, action, name):
```

Comment:
Should this be part of a different PR?

Reply:
Have to be done in this PR because it's a limitation of adding the support for `dots` in the name (as explained in my previous comment, you cannot do `cd /disks/<pool>/<image>`. So we have to move this command from `/disks/<disk_id>` to `/disks`.